requirements-ml.txt

```diff
@@ -1,7 +1,7 @@
 scikit-learn>=0.23.2
 keras>=2.4.3
 rapidfuzz>=2.6.1
-tensorflow-gpu>=2.6.4; sys.platform == 'linux'
+tensorflow-gpu <= 2.11; sys.platform == 'linux'
```

Comment:
I think this needs to be the combo, right?

Reply:
touche

Reply:
fixed and pushed